src/server/ns_turn_maps.c

```diff
@@ -201,7 +201,7 @@ int ur_map_foreach(ur_map* map, foreachcb_type func) {
   return 0;
 }
 
-int ur_map_foreach_arg(ur_map* map, foreachcb_arg_type func, void* arg) {
+int ur_map_foreach_arg(const ur_map* map, foreachcb_arg_type func, void* arg) {
```

Comment:
Do we want this in this PR? What does it fix?

Reply:
I had another approach how to add more metrics at first and missed to remove that. This only improves readability for new devs to understand which are in and which are out parameters.